modules/monitoring-installing-cluster-observability-operator-using-the-web-console.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+
+// * observability/monitoring/cluster_observability_operator/installing-the-cluster-observability-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="installing-the-cluster-observability-operator-in-the-web-console-_{context}"]
+= Installing the {coo-full} in the web console
+Install the {coo-first} from OperatorHub by using the {product-title} web console.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` cluster role.
+* You have logged in to the {product-title} web console.
+
+.Procedure
+
+. In the {product-title} web console, click *Operators* -> *OperatorHub*.
+. Click  *{coo-full}* in the list of available Operators in the *OperatorHub*.
```

Comment:
Better to change as the following
Search and click *{coo-full}* in the list of available Operators, then Install button

Reply:
Changed per your suggestion